www/_template/tutorials/svelte.md

```diff
@@ -0,0 +1,327 @@
+
+> Snowpack is … astonishingly fast, and has a beautiful development experience (hot module reloading, error overlays and so on), and we've been working closely with the Snowpack team on features like SSR[Server-side rendering]. The hot module reloading is particularly revelatory. - [Rich Harris, creator of Svelte](https://svelte.dev/blog/whats-the-deal-with-sveltekit)
+
+This guide is a step by step from an empty directory to a fully configured Snowpack project, in the process teaching:
+
+- How to set up your Snowpack development environment
+- Adding your first Svelte component
+- Importing images and other web assets
+- Enabling Hot Module Replacement (HMR)
+- Connecting your favorite tools
+
+Prerequisites: Snowpack is a command-line tool installed from npm. This guide assumes a basic understanding of Node.js, npm, and how to run commands in the terminal. Knowledge of Svelte is not required, Snowpack is an excellent way to learn Svelte!
+
+> 💡 Tip: a [Svelte/Snowpack](https://github.com/snowpackjs/snowpack/tree/master/create-snowpack-app/app-template-svelte) working example is available in our Create Snowpack App templates.
+
+## Getting started
+
+The easiest way to start a new Snowpack project is with [Create Snowpack App](<https://www.snowpack.dev/#create-snowpack-app-(csa)>), a tool for creating a new project based on our example templates. `@snowpack/app-template-minimal` is a Create Snowpack App template for a simple, bare-bones Snowpack project setup that the rest of this guide builds on.
+
+Run the following command in your terminal to create a new directory called `svelte-snowpack` with the minimal template installed:
+
+```bash
+npx create-snowpack-app svelte-snowpack --template @snowpack/app-template-minimal
+```
+
+Head to the new `svelte-snowpack` directory and start Snowpack with the following two commands:
+
+```bash
+cd svelte-snowpack
+npm run start
+```
+
+You should see your new website up and running!
+
+<div class="frame"><img src="/img/guides/react/minimalist-hello-world.png" alt="screenshot of project-template-minimal, which shows 'Hello world' in text on a white background." class="screenshot"/></div>
+
+Now that you have a basic project up and running! The next step is to install Svelte. Run the following command in your project directory:
+
+```bash
+npm install svelte --save
+```
+
+> 💡 Tip: add the "--use-yarn" or "--use-pnpm" flag to use something other than npm
+
+Snowpack [plugins](/plugins) are a way to extend Snowpack's capabilities without having to do custom configuration yourself. Install the `@snowpack/plugin-svelte` plugin so that Snowpack knows how built `.svelte` files into JavaScript and CSS files that run in the browser:
```

Comment:
I might move the `Snowpack [plugins](/plugins) are a way to extend Snowpack's capabilities without having to do custom configuration yourself` sentence to after the code snippet. I think it makes sense to show the step and then explain the step, vs. explain the larger concept first in a tutorial.